### DISCUSSION TITLE
Is it safe to put the code server with SSH login on the Internet with htacces?

### DISCUSSION THREAD

**Discussion by nvima** [coder/code-server#4321] [Q&A] [answered]
I have a question.
I would like to save my SSH connections in the CodeServer and make them globally accessible. I would also like to put the web server on the Internet via a domain in order to access it via WiFi at work or with a laptop on the go.
I know there is never 100% security, but is the server 99% secure if I provide it with htacces password in addition to the password and block bruteforce attack with fail2ban for example.
i am grateful for any suggestions and tips

**Comment by jsjoeio** [comment]
Great question! I am no security expert but maybe @bpmct @jawnsy or @code-asher has some thoughts/suggestions!

**Comment by bpmct** [comment, marked as answer]
I keep my code-server install on a public domain, but protected by a password. code-server --link does the same thing, but protected with GitHub log in.

As long as you use password protection, and it seems like you are using two, you should be fine :)
